app.go

```diff
@@ -404,6 +404,10 @@ func Serve() {
 	http.Handle("/", r)
 
 	// Start web application server
+	var bindAddress = app.cfg.Server.Bind
+	if bindAddress == "" {
+		bindAddress = "localhost"
+	}
 	if app.cfg.IsSecureStandalone() {
 		log.Info("Serving redirects on http://localhost:80")
```

Comment:
We'll probably want to include the `bindAddress` here too.

Reply:
Yeah, was in doubt about that one. pushed